README.md

```diff
@@ -234,11 +241,12 @@ const config = {
     postcss: () => {
     return [
       /* eslint-disable global-require */
-      require('postcss-cssnext')({
+      require('postcss-preset-env')({
+        stage: 0,
         features: {
-          customProperties: {
-            variables: reactToolboxVariables,
```

Comment:
Hi! I was giving this a last review before merging it and, aren't we losing the variables customization here?

Reply:
Yep, I went and added it back to the readme to showcase that the option is still there.

Reply:
Perfect! I'll try to merge the PR tonight